kubeflow/common/iap-ingress/base/swagger_template.yaml

```diff
@@ -0,0 +1,64 @@
+swagger: "2.0"
+info:
+  description: "wildcard config for any HTTP service."
+  title: "General HTTP Service."
+  version: "1.0.0"
+host: "ENDPOINT_NAME"
+x-google-endpoints:
+- name: "ENDPOINT_NAME"
+  target: "INGRESS_TARGET_IP"
+basePath: "/"
+consumes:
+- "application/json"
+produces:
+- "application/json"
+schemes:
+- "http"
+- "https"
+paths:
+  "/**":
+    get:
+      operationId: Get
+      responses:
+        '200':
+          description: Get
```

Comment:
To help understanding: Is this file used for describing that cloud endpoint accepts RESTful API?

Reply:
This is correct. This is a template describing [OpenAPI spec](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/2.0.md#specification) for a RESTful API

Reply:
Good finding! It will be great to include this specification in a README file.